### DISCUSSION TITLE
Call to graphql endpoint for non Spring boot application

### DISCUSSION THREAD

**Discussion by ABSringar** [graphql-java/graphql-java#2617] [Q&A]
I have been provided graphql enpoint! I need to create query and execute from my java non spring boot application. Is there a dependency where i can add in my maven project and make a graphql call  ? Could any one share me the sample code ?

I'm using AEM cms and sling API!

**Comment by phillip-kruger** [comment]
I assume the endpoint is via HTTP ? You can try and use SmallRye GraphQL Client. There are two options: 

- Dynamic client (https://github.com/smallrye/smallrye-graphql/blob/main/client/README-dynamic-client.adoc)
- Typesafe client (https://github.com/smallrye/smallrye-graphql/blob/main/client/README-typesafe-client.adoc)

These are primarily used in Quarkus, but should be usable standalone.
